src/main/java/io/appium/java_client/ios/PerformsTouchID.java

```diff
@@ -35,9 +35,24 @@ default void performTouchID(boolean match) {
 
     /**
      * Enrolls touchId in iOS Simulators.
-     *
+     * This method is deprecated. Please use {@link #toggleTouchIDEnrollment(boolean)} instead
      */
+    @Deprecated
     default void toggleTouchIDEnrollment() {
```

Comment:
marking it as deprecated is safer. We can remove as soon as beta is released (what we, actually, always do)